WebInstall/explainer_cross_domain.md

```diff
@@ -10,37 +10,57 @@ This document is a starting point for engaging the community and standards bodie
 
-While this is the general acquisition flow on many platforms, the web does not have the ability to replicate this scenario because it can not install applications. This makes it impossible for a web app (repository, catalog, store) to install and distribute other applications.
+The acquisition of an web app can originate via multiple user flows that generally involve search, navigation, and ultimately trust that the UA will prompt or provide some sort of UX that support "[installing](https://web.dev/articles/install-criteria)/[adding](https://support.apple.com/en-gb/guide/safari/ibrw9e991864/mac)" the desired app. There are multiple use cases for this feature, as seen in the [use case](#use-cases) section, but *the core concept is the installation of a web app directly from a web page*.
+
+The current alternative to the Web Install API implies the user needs to rely on search engines, app stores, proprietary protocols, proprietary "smart" banners, UA prompts, hidden UX and other means that take the user out of their navigating context. They also represent additional steps towards the acquisition of the app.
+
+Inherently, these alternative user flows to "install" an app rely on multi-step processes that at best require a couple of clicks to navigate to an origin and install it, and at worst involve the user searching on browser menus for a way to add the app to their device. The Web platform is not capable of providing a seamless, consistent experience that allows users to discover and acquire applications in a frictionless manner. Every additional step in the acquisition funnel for web apps comes with an additional drop off rate as well. 
+
+Moreover, the **Web Install API feature is beneficial for app discovery**: it allows developers to create their own acquisition mechanism and tailor it to benefit users that:
+* might not know that a web app exists for the current origin.
+* don't understand what the icon/prompt in the omnibox does.
+* don't know how to deep search several layers of browser UX to add the app to their devices.
+* don't use app stores to discover new app experiences.
 
-**The Web Install API** addresses this shortcoming in the platform. It **allows a web site to install a web app *([same](./explainer_same_domain.md) or **cross** origin)*. This functionality allows the creation of web based catalogs that can install PWAs directly from the web and into multiple platforms.**
 
 ## Goals
 
 * **Enable installation of web apps (cross-origin).**
-* Allow a ***vetted* installation origin** to know if the web app is installed (see *[`install_sources`](#install-sources-manifest-field) new manifest field*).
+
```

Comment:
This section no longer displays with bullet points. I assume that's unintentional?

Reply:
@amandabaker not sure what you mean? I see bullet points here in the goals?

Reply:
They show as deleted for me. I wonder if it's due to the extra whitespace above?
![image](https://github.com/user-attachments/assets/540e3443-7be0-47f3-be8f-6c6c06a5b931)